crimson/verifybamid.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+    crimson.verifybamid
+    ~~~~~~~~~~~~~~
```

Comment:
Minor style comment: it's nicer if the tildes have the same amount of characters as the preceeding line.